app/src/debug/java/timur/gilfanov/messenger/debug/DebugDataRepository.kt

```diff
@@ -0,0 +1,15 @@
+package timur.gilfanov.messenger.debug
+
+import kotlinx.coroutines.flow.Flow
+
+interface DebugDataRepository {
+
+    suspend fun getSavedScenario(): DataScenario?
+    suspend fun toggleAutoActivity(enabled: Boolean)
+    suspend fun toggleNotification(show: Boolean)
```

Comment:
If it's toggle then no need to pass a new value. Must use `set` here if funtions need new values to be passed.
```suggestion
    suspend fun setAutoActivity(enabled: Boolean)
    suspend fun setNotification(show: Boolean)
```